software/opensign.yml

```diff
@@ -0,0 +1,12 @@
+name: "OpenSign"
+website_url: "https://www.opensignlabs.com"
+source_code_url: "https://github.com/opensignlabs/opensign"
+description: "Free, open source & self-hosted DocuSign alternative."
```

Comment:
The description should explain what the software does (you can add `(alternative to DocuSign)` at the end of the description)